src/Common/MetricHelper.cs

```diff
@@ -425,7 +425,7 @@ private void PopulatePropertiesFromQos(AzurePSQoSEvent qos, IDictionary<string,
         private static string[] exceptionTrackAcceptModuleList = { "Az.Accounts", "Az.Compute", "Az.AKS", "Az.ContainerRegistry" };
         private static string[] exceptionTrackAcceptCmdletList = { "Get-AzKeyVaultSecret", "Get-AzKeyVaultCert" };
 
-        private static string ConvertFrameToString(StackFrame frame)
+        private static string ConvertFrameToString(System.Diagnostics.StackFrame frame)
```

Comment:
Change on this line doesn't seem necessary

Reply:
StackFrame also defined in new version of Application Insights namespace and so I add prefix here.